share/completions/funcsave.fish

```diff
@@ -1 +1,2 @@
 complete -c funcsave -xa "(functions -na)" -d "Save function"
+complete -c funcsave -s d -l directory -d "Directory to save the functions" -r -F
```

Comment:
I would just complete $fish_function_path here.

Saving a function to a directory that's not in there isn't likely to be helpful, and completing *files* is even less so.

Reply:
I thought the `-F` options also worked for path.

Reply:
Well, yes, it does. But it also works for every other directory, and worse every other file.

`funcsave --directory ~` is unlikely to be what you want (as you then can't actually use the function) and `funcsave --directory /usr/bin/grep` is complete nonsense. So it's unhelpful to complete either of these.